src/reportseff/db_inquirer.py

```diff
@@ -438,7 +457,7 @@ def get_partition_timelimits(self) -> dict:
         Raises:
             RuntimeError: if scontrol raises an error
         """
-        command_args = "scontrol show partition".split()
+        command_args = f"scontrol {self.extra_args} show partition".split()
```

Comment:
My memory may be foggy, but it seems like the cluster argument won't get forwarded here, which is what #44 was having an error with.  Currently cluster seems to only affect the sacct args.  Maybe some of that function should be refactored to a private method to add in extra args and cluster which can be used here and in set_sacct_args?

Reply:
I fix this. when `--cluster` is provided it will call `scontrol --cluster cluster show partition` otherwise stay the same